File: tests/machine/x/dart/match_expr.dart
var x = 2;

var label = (() {
  var _t = x;
  if (_t == 1) {
    return 'one';
  } else if (_t == 2) {
    return 'two';
  } else if (_t == 3) {
    return 'three';
  } else {
    return 'unknown';
  }
  return null;
})();

void main() {
  print(label);
}

File: tests/machine/x/rb/match_expr.rb
x = 2
label = (begin
	_t0 = x
	case
	when _t0 == 1
		"one"
	when _t0 == 2
		"two"
	when _t0 == 3
		"three"
	else
		"unknown"
	end
end)
puts(label)
